examples/hf/pippy_albert.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+
+# Minimum effort to run this example:
+# $ torchrun --nproc-per-node 4 pippy_albert.py
+
+import argparse
+import os
+
+import torch
+import torch.distributed as dist
+
+from pippy.IR import Pipe, PipeSplitWrapper, annotate_split_points
+from pippy.PipelineStage import PipelineStage
+
+from transformers import AlbertForMaskedLM, AlbertConfig
+
+from hf_utils import generate_inputs_for_model, get_number_of_params
```

Comment:
I can't locate the hf_utils file? 
I don't find it in the repo - am I missing an install or was this meant to be added?

Reply:
I found it - it's in another PR:
#875